app/controllers/posts_controller.rb

```diff
@@ -20,7 +20,8 @@ def create
 	end
 
   def index
-    @posts = current_user.posts.order(:created_at).page(params[:page])
+    # @posts = current_user.posts.order(:created_at).page(params[:page])
+    @posts = Post.where(user_id: params[:id]).order(:created_at).page(params[:page])
```

Comment:
Please try to remove lines instead of commenting and make a good habit of self-reviewing the code you push